Prison Life Visualization/prisonlife/Scene/main_level_4.tscn

```diff
@@ -878,9 +878,9 @@ scale = Vector2(0.84944, 0.610442)
 position = Vector2(320, 361)
 scale = Vector2(0.84944, 0.690442)
 
-[node name="previous" parent="." instance=ExtResource("14_jeo7u")]
-position = Vector2(25, 563)
-previousLevel = ExtResource("15_q3m4s")
+; [node name="previous" parent="." instance=ExtResource("14_jeo7u")]
```

Comment:
@LEADGREY This is specifically the chunk that was causing the problem. I commented it out of the scene file and that did the trick